cmd/katapult/data_centers.go

```diff
@@ -15,28 +14,28 @@ type dataCentersClient interface {
 	Get(ctx context.Context, ref core.DataCenterRef) (*core.DataCenter, *katapult.Response, error)
 }
 
+const dataCentersFormat = "{{ range $dc := . }}" +
```

Comment:
Perhaps it would be good practice for us to always embed files for these templates ?